[FEATURE] Future events (#3323)
ClanGenOfficial/clangen: docs/dev/writing/shortevents.md

| "male"                                      | Makes the cat male.                                                                                                                                                                                                                                                                                                                                                 |
| "female"                                    | Makes the cat female.                                                                                                                                                                                                                                                                                                                                               |

> | "can_birth"                                 | If same-sex breeding is OFF, make the cat female. Otherwise, 50 - 50 gender-roll.                                                                                                                                                                                                                                                                                   |

Reviewer: ```suggestion
| "can_birth"                                 | If same-sex breeding is OFF, make the cat female. Otherwise, either male or female will be chosen at random.                                                                                                                                                                                                                                                                                   |
```

| "new_name"                                  | Ensure the cat takes on a clan-like name.                                                                                                                                                                                                                                                                                                                           |
| "old_name"                                  | Ensure the cat keeps their old (maybe loner or kittypet) name. Doesn't work for kittens or litters.                                                                                                                                                                                                                                                                 |

> | "kittypet"                                  | Gives the cat a kitty-pet type backstory. If "meeting" is also included, this tag will make the cat an kittypet outsider.                                                                                                                                                                                                                                           |

Reviewer: ```suggestion
| "kittypet"                                  | Gives the cat a kitty-pet type backstory. If "meeting" is also included, this tag will make the cat a kittypet outsider.                                                                                                                                                                                                                                           |
```

> | "loner"                                     | Gives the cat a loner type backstory. If "meeting" is also included, this tag will make the cat an loner outsider.                                                                                                                                                                                                                                                  |

Reviewer: ```suggestion
| "loner"                                     | Gives the cat a loner type backstory. If "meeting" is also included, this tag will make the cat a loner outsider.                                                                                                                                                                                                                                                  |
```

> | "rogue"                                     | Gives the cat a rogue type backstory. If "meeting" is also included, this tag will make the cat an rogue outsider.                                                                                                                                                                                                                                                  |

Reviewer: ```suggestion
| "rogue"                                     | Gives the cat a rogue type backstory. If "meeting" is also included, this tag will make the cat a rogue outsider.                                                                                                                                                                                                                                                  |
```

| "clancat"                                   | Gives the cat a former-clancat type backstory. If "meeting" is also included, this tag will make the cat a former Clancat outsider.                                                                                                                                                                                                                                 |

> | "meeting"                                   | Make the cat an outsider (the patrol just met them, but they didn't join). That cat will never take a new clan-like name.                                                                                                                                                                                                                                           |

Reviewer: ```suggestion
| "meeting"                                   | Make the cat an outsider (the patrol just met them, but they didn't join). That cat will never take a new Clan-like name.                                                                                                                                                                                                                                           |
```

> | "exists"                                    | Will attempt to find an existing outsider to utilize instead of creating a new one.  Keep in mind that this ONLY checks if the existing outsider matches indicated status, age, and backstory.  DO NOT use this tag in conjunction with creating litters or assigning parentage.                                                                                    |

Reviewer: ```suggestion
| "exists"                                    | Will attempt to find an existing outsider to utilize instead of creating a new one. Keep in mind that this ONLY checks if the existing outsider matches indicated status, age, and backstory.  DO NOT use this tag in conjunction with creating litters or assigning parentage.                                                                                    |
```
